test/cookbooks/sensu-test/recipes/gem_lwrp.rb

```diff
@@ -10,7 +10,7 @@
 
 # ensure we pass the specified version
 sensu_gem 'sensu-plugins-chef' do
-  version '0.0.5'
+  version '2.0.0'
```

Comment:
Lets bump this to newest that was re-written with ridley and does not use the deprecated apis that were still present in 2.x